README.md

```diff
@@ -80,25 +123,18 @@ $ captplanet setup
 $ captplanet run
 ```
 
-### Configure AWS CLI
+### Try out some commands via Storj CLI or AWS CLI
 
```

Comment:
Maybe add a link to some aws commands?

Reply:
we have AWS commands listed above and a link to their CLI doc site 
https://docs.aws.amazon.com/cli/latest/reference/s3/index.html